.github/workflows/check.yml

```diff
@@ -7,6 +7,17 @@ on:
       - main
   # ..and any pull request.
   pull_request:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Run Checks'
+  # Cron job to run checks @ 8:30 pm daily on the latest commit on the default branch - main
```

Comment:
why 8:30pm?  is that significant?

Reply:
I had to pick a time and currently GitHub actions don't have [time zone support](https://github.com/orgs/community/discussions/13454). As this is in UTC This should be around 1:30 pm PST for most of the year.